doc/articles/getting-started/wizard/projects.md

```diff
@@ -0,0 +1,8 @@
+---
+uid: Uno.GettingStarted.UsingWizard.Projects
+---
+
+- **Server**  
+Adds an ASP.NET Core Server projects to the solution, which hosts the WASM project, and can also be used to create an API and endpoints. It can also be used as the data server as well as the authentication server. At this stage Authentication is not implemented in the server project, but this feature is being considered of being shipped with the template to reduce boilerplate code even further.
```

Comment:
Remove sentence about Auth not being in server project - don't talk about future features as this makes a commitment for us to ship that